### PR TITLE
Tcs 21 include the filter that is being used in the return payload

### PR DESCRIPTION
**Pull request by MateaKanjuga**

### CODE REVIEW

**Review by jwrno1 — APPROVED** [review]

File: code/segmentedtickerplant/datastripe.q [hunk 1]
```diff
@@ -45,14 +45,10 @@ subsegment:{[tbl;segid];
      //tablename and segmentid used to get filters
      if[tbl~`;:.z.s[;segid] each .stpps.t];
      if[not tbl in .stpps.t;
-          .lg.e[`sub;m:"Table ",string[tbl]," not in list of stp pub/sub tables"];
-          :(tbl;m)
```
Comment (rian-morgan) [inline code/segmentedtickerplant/datastripe.q:49]: This should return an empty list. 
`:(tbl;m) -> :()`

File: code/segmentedtickerplant/datastripe.q [hunk 2]
```diff
@@ -45,14 +45,10 @@ subsegment:{[tbl;segid];
...
-     ];
+          .lg.e[`sub;m:"Table ",string[tbl]," not in list of stp pub/sub tables"]];
      filter:segmentfilter[tbl;segid];
      if[filter~"";
-          .lg.e[`sub;m:"Incorrect pairing of table ",string[tbl]," and segmentID ",string[segid]," not found in .stpps.segmentconfig"];
-          :(tbl;m)
```
Comment (rian-morgan) [inline code/segmentedtickerplant/datastripe.q:54]: This should return an empty list. 
`:(tbl;m) -> :()`

File: code/segmentedtickerplant/datastripe.q [hunk 3]
```diff
@@ -25,13 +25,13 @@ initdatastripe:{
...
-// Function to map the where clause from config table extracted by .stpps.segmentfilter function to tablename
-// Allows use of ` as argument for tables
-
 \d .stpps
 
+// Function to map the where clause from config table extracted by .stpps.segmentfilter function to tablename
+// Allows use of ` as argument for tables
 filtermap:{[tabs;id] if[tabs~`;tabs:.stpps.t]; ((),tabs)!.stpps.segmentfilter\:[(),tabs;id]}
 
+
```
Comment (rian-morgan) [inline code/segmentedtickerplant/datastripe.q:34]: remove this line

**Review by rian-morgan — CHANGES_REQUESTED** [review]

**Comment by rian-morgan** [comment]
Resolve conflicts before merge please

**Review by jwrno1 — APPROVED** [review]

File: code/segmentedtickerplant/datastripe.q [hunk 4]
```diff
@@ -52,22 +51,21 @@ subsegment:{[tbl;segid];
...
 
 // the subdetails function adapted to also retrieve filters from the segmented tickerplant
 segmentedsubdetails: {[tabs;instruments;segid] (!). flip 2 cut (
-     `schemalist ; .stpps.subsegment\:[tabs;segid];
-     `logfilelist ; .stplg.replaylog[tabs];
-     `rowcounts ; tabrowcounts[tabs];
-     `date ; (.eodtime `d);
-     `logdir ; `$getenv`KDBTPLOG;
-     `filters ; .stpps.filtermap[tabs;segid]
-     )}
```
Comment (rian-morgan) [inline code/segmentedtickerplant/datastripe.q:68]: this is broken!

**Review by rian-morgan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**